tests/test_views.py

```diff
@@ -4140,6 +4145,30 @@ def test_creates_a_project(self):
         assert_true(node)
         assert_true(node.title, 'Im a real title')
 
+    def test_create_component_add_contributors(self):
+        url = web_url_for('project_new_node', pid=self.project._id)
+        post_data = {'title': 'New Component With Contributors Title',  'category': '', 'inheritContributors': 'true'}
+        request = self.app.post(url, post_data, auth=self.user1.auth)
+        self.project.reload()
+        child = self.project.nodes[0]
+        assert_equal(child.title, 'New Component With Contributors Title')
+        assert_in(self.user1, child.contributors)
+        assert_in(self.user2, child.contributors)
+        # check redirect url to make sure its not going to pid/components
+        assert_in('/contributors/', request)
```

Comment:
Judging by the comment, I don't think this assertion is doing what you expect; `assert_in('foo', res)` will check that the text 'foo' is anywhere in the response body. 

You should be able to access the request URL more directly. IIRC, it's something like `res.location` or `res.request.path`.